### ISSUE TITLE
__docker_daemon_is_experimental: command not found

### ISSUE THREAD

**Issue by pszczekutowicz** [docker/docker#30858]
**Description**

Bash completion for docker service shows __docker_daemon_is_experimental: command not found

Ubuntu 16.10, Docker Engine 1.13.1

**Steps to reproduce the issue:**
1. in terminal type: docker service
2. press tab

**Result:**
__docker_daemon_is_experimental: command not found

**Expected:**
List of available commands should be printed.

**Output of `docker version`:**

```
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:57:21 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:57:21 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Output of `docker info`:**

```
Containers: 8
 Running: 2
 Paused: 0
 Stopped: 6
Images: 73
Server Version: 1.13.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 153
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: active
 NodeID: dxtd2cwx56h2rnt3m0l26fzqm
 Is Manager: true
 ClusterID: 3tk9zsy96qd1cbnscb94cpp84
 Managers: 1
 Nodes: 1
 Orchestration:
  Task History Retention Limit: 5
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: 3 months
 Node Address: 10.57.206.96
 Manager Addresses:
  10.57.206.96:2377
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1
runc version: 9df8b306d01f59d3a8029be411de015b7304dd8f
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.8.0-37-generic
Operating System: Ubuntu 16.10
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 15.54 GiB
Name: ********************
ID: G244:BV5P:3LAX:PHPJ:DVPH:GHYC:RC2X:AZLG:JSTN:EQEI:HLVP:W7LL
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Comment by AkihiroSuda** [comment]
@albers PTAL?

**Comment by albers** [comment]
@pszczekutowicz Please post the output of `ls -l /etc/bash_completion.d/docker` and `md5sum /etc/bash_completion.d/docker`.

**Comment by albers** [comment]
@pszczekutowicz Did you do a clean install or an upgrade of Docker?
If you upgraded, please tell me your upgrade history, beginning from the last 12.x release.

**Comment by albers** [comment]
I can reproduce, will investigate further.

**Comment by pszczekutowicz** [comment]
Sorry, I was unable to respond earlier. Thanks for the fix.

**Comment by albers** [comment]
@pszczekutowicz The issue will be fixed in a potential 1.13.2 release.
See https://github.com/docker/docker/pull/30875#issuecomment-279323995 for how to get a fixed bash completion script.
